### PR TITLE
Allow dependency on symfony/framework-bundle:^3.0

### PR DESCRIPTION
**Pull request by trq**

### CODE REVIEW

File: Resources/config/services.xml [hunk 1]
```diff
@@ -11,10 +11,8 @@
 
     <services>
 
-        <service id="jcroll_foursquare_api.foursquare_client"
-                 class="Jcroll\FoursquareApiClient\Client\FoursquareClient"
-                 factory-class="Jcroll\FoursquareApiClient\Client\FoursquareClient"
-                 factory-method="factory">
+        <service id="jcroll_foursquare_api.foursquare_client" class="Jcroll\FoursquareApiClient\Client\FoursquareClient">
+            <factory class="Jcroll\FoursquareApiClient\Client\FoursquareClient" method="factory" />
```
Comment (jcroll) [inline Resources/config/services.xml:15]: Hi @trq thanks for your contribution.

I believe the above factory pattern was introduced in symfony 2.6 so below it should be:
`"symfony/framework-bundle": "~2.6|^3.0"`

After that we are good for merge and much appreciated.

**Comment by jcroll** [comment]
Additionally if you can please squash to one commit.

**Comment by trq** [comment]
Done, thanks.

**Comment by jcroll** [comment]
many thanks @trq

**Pull request merged**